kolena/dataset/dataset.py

```diff
@@ -156,20 +125,10 @@ def _infer_datatype(df: pd.DataFrame) -> Union[pd.DataFrame, str]:
     return DatapointType.TABULAR.value
 
 
-def _infer_id_fields(df: pd.DataFrame) -> List[str]:
-    def get_id_fields_by(field: str) -> List[str]:
-        return [
-            id_field
-            for id_field in df.columns.array
-            if isinstance(id_field, str) and id_field.rsplit(_SEP, maxsplit=1)[-1] == field
-        ]
-
-    if id_fields := get_id_fields_by(_FIELD_ID):
-        return id_fields
-    elif id_fields := get_id_fields_by(_FIELD_LOCATOR):
-        return id_fields
-    elif id_fields := get_id_fields_by(_FIELD_TEXT):
-        return id_fields
+def _infer_id_field(df: pd.DataFrame) -> str:
+    for field in [_FIELD_ID, _FIELD_LOCATOR, _FIELD_TEXT]:
+        if field in df.columns:
+            return field
     raise InputValidationError("Failed to infer the id_fields, please provide id_fields explicitly")
```

Comment:
what's the drawback of keeping the original func name and typing signature?
i think return list[str] will be consistent with our `id_fields` defination, and we can use this function as entrypoint for any future extending